sandpack-react/src/contexts/utils/useClient.ts

```diff
@@ -159,6 +162,26 @@ export const useClient: UseClient = (
         }, timeOut);
       }
 
+      const getStableServiceWorkerId = async () => {
+        const key = `SANDPACK_INTERNAL:URL-CONSISTENT-ID`;
+        const fixedId = localStorage.getItem(key);
+        if (fixedId) {
```

Comment:
Maybe we should do `localStorage.setItem` for the id if it turns out it's not set (with a random id). Then we don't have to ask the user to set it.